cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -316,7 +323,9 @@ private BuildOptions constructBuildOptions() {
                 .setNativeImage(nativeImage)
                 .disableSyntaxTreeCaching(disableSyntaxTreeCaching)
                 .setGraalVMBuildOptions(graalVMBuildOptions)
-                .setShowDependencyDiagnostics(showDependencyDiagnostics);
+                .setShowDependencyDiagnostics(showDependencyDiagnostics)
+                .setOptimizeCodegen(optimizeCodegen)
+                .setVerbose(verbose);
```

Comment:
We allow setting build options in the Ballerina.toml too. These changes has to be added to https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/ManifestBuilder.java#L658

Reply:
Fixed in 1a166f6c99b430d241f9a4a7dc602a898006fcff.